doc/cpp/interconnectivity.rst

```diff
@@ -37,15 +35,17 @@ Interconnectivity
 .. cpp:class:: gap_junction_connection
 
     Describes a gap junction between two gap junction sites.
-    Gap junction sites are represented by :cpp:type:cell_member_type.
-
-    .. cpp:member:: cell_member_type local
 
-        gap junction site: one half of the gap junction connection.
+    A :class:`gap_junction_connection` is associated with the local cell of the connection and
+    its gid.
```

Comment:
I'm not sure what this sentence means — would it be possible to rephrase it?

Reply:
This was my attempt at clarifying why we don't need the `gid` of the `local` cell anymore. Connections used to be "complete" by themselves, now they're only complete in the context of the recipe, when they are bound to a cell. I can remove it, or I can replace it with:
> Describes a gap junction between two gap junction sites. The `local` site does not include the gid of a cell, this is because a `gap_junction_connection` is bound to the local cell which means that the `gid` is implicitly known.

What do you think?

Reply:
Also I should probably clarify that to create a bidirectional gap junction, 2 gap junction connections are needed. I'll add that to the concepts/cpp/python pages

Reply:
I think the revised statement is very clear, thanks!